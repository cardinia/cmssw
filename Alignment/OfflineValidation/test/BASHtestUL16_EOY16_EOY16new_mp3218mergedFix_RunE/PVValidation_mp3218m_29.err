27-Sep-2019 21:43:07 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/2E1A6712-E207-E911-87C9-008CFAC93E04.root
27-Sep-2019 21:43:10 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/2E1A6712-E207-E911-87C9-008CFAC93E04.root
Begin processing the 1st record. Run 277127, Event 1354984232, LumiSection 720 on stream 0 at 27-Sep-2019 21:43:25.203 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 21:47:34 CEST  Run: 277127 Event: 1362068854
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
Begin processing the 1001st record. Run 277127, Event 1362322444, LumiSection 724 on stream 0 at 27-Sep-2019 21:47:48.753 CEST
Begin processing the 2001st record. Run 277127, Event 1369857600, LumiSection 728 on stream 0 at 27-Sep-2019 21:51:59.979 CEST
Begin processing the 3001st record. Run 277127, Event 1378230850, LumiSection 733 on stream 0 at 27-Sep-2019 21:55:53.512 CEST
Begin processing the 4001st record. Run 277127, Event 1387098697, LumiSection 738 on stream 0 at 27-Sep-2019 21:59:59.880 CEST
Begin processing the 5001st record. Run 277127, Event 1394425626, LumiSection 742 on stream 0 at 27-Sep-2019 22:04:08.532 CEST
Begin processing the 6001st record. Run 277127, Event 1403720529, LumiSection 747 on stream 0 at 27-Sep-2019 22:08:19.304 CEST
Begin processing the 7001st record. Run 277127, Event 1411622539, LumiSection 752 on stream 0 at 27-Sep-2019 22:12:21.402 CEST
Begin processing the 8001st record. Run 277127, Event 1418581178, LumiSection 756 on stream 0 at 27-Sep-2019 22:16:37.070 CEST
Begin processing the 9001st record. Run 277127, Event 1428086961, LumiSection 761 on stream 0 at 27-Sep-2019 22:20:55.148 CEST
Begin processing the 10001st record. Run 277127, Event 1434495801, LumiSection 765 on stream 0 at 27-Sep-2019 22:25:08.666 CEST
Begin processing the 11001st record. Run 277127, Event 1442306455, LumiSection 770 on stream 0 at 27-Sep-2019 22:29:09.851 CEST
Begin processing the 12001st record. Run 277127, Event 1451990910, LumiSection 775 on stream 0 at 27-Sep-2019 22:33:09.402 CEST
Begin processing the 13001st record. Run 277127, Event 1459029516, LumiSection 779 on stream 0 at 27-Sep-2019 22:36:48.822 CEST
Begin processing the 14001st record. Run 277127, Event 1467827573, LumiSection 784 on stream 0 at 27-Sep-2019 22:40:39.164 CEST
Begin processing the 15001st record. Run 277127, Event 1476469381, LumiSection 789 on stream 0 at 27-Sep-2019 22:44:35.330 CEST
Begin processing the 16001st record. Run 277127, Event 1483430594, LumiSection 793 on stream 0 at 27-Sep-2019 22:48:20.719 CEST
Begin processing the 17001st record. Run 277127, Event 1492328268, LumiSection 798 on stream 0 at 27-Sep-2019 22:52:25.283 CEST
Begin processing the 18001st record. Run 277127, Event 1498055398, LumiSection 802 on stream 0 at 27-Sep-2019 22:56:28.351 CEST
Begin processing the 19001st record. Run 277127, Event 1506228028, LumiSection 807 on stream 0 at 27-Sep-2019 23:00:26.876 CEST
Begin processing the 20001st record. Run 277127, Event 1516443847, LumiSection 812 on stream 0 at 27-Sep-2019 23:04:31.553 CEST
Begin processing the 21001st record. Run 277127, Event 1523960225, LumiSection 817 on stream 0 at 27-Sep-2019 23:08:12.963 CEST
Begin processing the 22001st record. Run 277127, Event 1533002521, LumiSection 822 on stream 0 at 27-Sep-2019 23:12:13.849 CEST
Begin processing the 23001st record. Run 277127, Event 1541450936, LumiSection 827 on stream 0 at 27-Sep-2019 23:16:00.141 CEST
Begin processing the 24001st record. Run 277127, Event 1547192124, LumiSection 831 on stream 0 at 27-Sep-2019 23:19:25.495 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 23:21:49 CEST  Run: 277127 Event: 1552729830
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 1
%MSG
Begin processing the 25001st record. Run 277127, Event 1557124122, LumiSection 836 on stream 0 at 27-Sep-2019 23:23:06.245 CEST
Begin processing the 26001st record. Run 277127, Event 1565096200, LumiSection 841 on stream 0 at 27-Sep-2019 23:26:53.636 CEST
Begin processing the 27001st record. Run 277127, Event 1573924976, LumiSection 846 on stream 0 at 27-Sep-2019 23:30:37.695 CEST
Begin processing the 28001st record. Run 277127, Event 1579336440, LumiSection 850 on stream 0 at 27-Sep-2019 23:34:16.884 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 23:36:32 CEST  Run: 277127 Event: 1585717462
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 74
%MSG
Begin processing the 29001st record. Run 277127, Event 1588696183, LumiSection 855 on stream 0 at 27-Sep-2019 23:37:47.300 CEST
Begin processing the 30001st record. Run 277127, Event 1597604655, LumiSection 859 on stream 0 at 27-Sep-2019 23:41:33.304 CEST
Begin processing the 31001st record. Run 277127, Event 1603504905, LumiSection 862 on stream 0 at 27-Sep-2019 23:45:32.810 CEST
Begin processing the 32001st record. Run 277127, Event 1611730290, LumiSection 866 on stream 0 at 27-Sep-2019 23:49:36.546 CEST
Begin processing the 33001st record. Run 277127, Event 1621195682, LumiSection 870 on stream 0 at 27-Sep-2019 23:53:27.691 CEST
27-Sep-2019 23:56:16 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/2E1A6712-E207-E911-87C9-008CFAC93E04.root
27-Sep-2019 23:56:16 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/8C1385E6-8407-E911-9504-002590D9D8C4.root
27-Sep-2019 23:56:18 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/8C1385E6-8407-E911-9504-002590D9D8C4.root
Begin processing the 34001st record. Run 277127, Event 1627050064, LumiSection 873 on stream 0 at 27-Sep-2019 23:57:24.512 CEST
Begin processing the 35001st record. Run 277127, Event 1636120602, LumiSection 877 on stream 0 at 28-Sep-2019 00:01:16.431 CEST
Begin processing the 36001st record. Run 277127, Event 1643501242, LumiSection 881 on stream 0 at 28-Sep-2019 00:05:02.749 CEST
Begin processing the 37001st record. Run 277127, Event 1652233982, LumiSection 885 on stream 0 at 28-Sep-2019 00:08:44.538 CEST
Begin processing the 38001st record. Run 277127, Event 1659552754, LumiSection 889 on stream 0 at 28-Sep-2019 00:12:46.184 CEST
Begin processing the 39001st record. Run 277127, Event 1666027356, LumiSection 892 on stream 0 at 28-Sep-2019 00:16:56.319 CEST
Begin processing the 40001st record. Run 277127, Event 1675337344, LumiSection 896 on stream 0 at 28-Sep-2019 00:20:52.111 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 28-Sep-2019 00:21:16 CEST  Run: 277127 Event: 1676391572
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
Begin processing the 41001st record. Run 277127, Event 1683516498, LumiSection 900 on stream 0 at 28-Sep-2019 00:24:40.633 CEST
28-Sep-2019 00:26:47 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/8C1385E6-8407-E911-9504-002590D9D8C4.root
28-Sep-2019 00:26:47 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/2C14F426-8507-E911-B66F-008CFAEBDC04.root
28-Sep-2019 00:26:49 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/2C14F426-8507-E911-B66F-008CFAEBDC04.root
Begin processing the 42001st record. Run 277127, Event 531696744, LumiSection 284 on stream 0 at 28-Sep-2019 00:28:27.480 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 28-Sep-2019 00:28:51 CEST  Run: 277127 Event: 534753306
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 2478
%MSG
Begin processing the 43001st record. Run 277127, Event 552680782, LumiSection 296 on stream 0 at 28-Sep-2019 00:32:20.476 CEST
Begin processing the 44001st record. Run 277127, Event 560076248, LumiSection 300 on stream 0 at 28-Sep-2019 00:36:23.127 CEST
Begin processing the 45001st record. Run 277127, Event 569053290, LumiSection 305 on stream 0 at 28-Sep-2019 00:40:20.802 CEST
Begin processing the 46001st record. Run 277127, Event 578312788, LumiSection 310 on stream 0 at 28-Sep-2019 00:44:26.178 CEST
Begin processing the 47001st record. Run 277127, Event 586185725, LumiSection 315 on stream 0 at 28-Sep-2019 00:47:54.352 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 28-Sep-2019 00:50:03 CEST  Run: 277127 Event: 592122540
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 1716
%MSG
Begin processing the 48001st record. Run 277127, Event 592783265, LumiSection 319 on stream 0 at 28-Sep-2019 00:51:12.548 CEST
Begin processing the 49001st record. Run 277127, Event 601338775, LumiSection 324 on stream 0 at 28-Sep-2019 00:54:27.378 CEST
28-Sep-2019 00:58:12 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/2C14F426-8507-E911-B66F-008CFAEBDC04.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 OutOfBounds          -w TrackRefitter:Fi                       6        6
    2 fileAction           -s file_close                             3        3
    3 fileAction           -s file_open                              6        6

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 OutOfBounds          277127/136206885 277127/155272983 277127/592122540
    2 fileAction           PostProcessEvent PostProcessEvent PostGlobalEndRun
    3 fileAction           pre-events       pre-events       PostProcessEvent

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 6                   6
System                  9                   9

dropped waiting message count 0
[9.849MB/9.849MB][100%][==================================================][9.849MB/s]  [9.849MB/9.849MB][100%][==================================================][9.849MB/s]  
