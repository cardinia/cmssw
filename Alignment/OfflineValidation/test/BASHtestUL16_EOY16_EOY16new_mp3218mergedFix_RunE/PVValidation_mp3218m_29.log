000 (3196552.029.000) 09/27 21:32:53 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3196552.029.000) 09/27 21:42:56 Started transferring input files
	Transferring to host: <188.184.183.48:9618?addrs=188.184.183.48-9618+[2001-1458-301-2b--100-126]-9618&noUDP&sock=5138_ef8a_13282>
...
040 (3196552.029.000) 09/27 21:42:56 Finished transferring input files
...
001 (3196552.029.000) 09/27 21:42:56 Job executing on host: <188.184.183.48:9618?addrs=188.184.183.48-9618+[2001-1458-301-2b--100-126]-9618&noUDP&sock=3293_4d75_3>
...
006 (3196552.029.000) 09/27 21:43:05 Image size of job updated: 205264
	114  -  MemoryUsage of job (MB)
	116436  -  ResidentSetSize of job (KB)
...
006 (3196552.029.000) 09/27 21:48:05 Image size of job updated: 658540
	539  -  MemoryUsage of job (MB)
	551584  -  ResidentSetSize of job (KB)
...
006 (3196552.029.000) 09/27 21:58:07 Image size of job updated: 661092
	542  -  MemoryUsage of job (MB)
	554136  -  ResidentSetSize of job (KB)
...
006 (3196552.029.000) 09/27 23:58:19 Image size of job updated: 680000
	561  -  MemoryUsage of job (MB)
	574248  -  ResidentSetSize of job (KB)
...
006 (3196552.029.000) 09/28 00:03:20 Image size of job updated: 680512
	562  -  MemoryUsage of job (MB)
	574760  -  ResidentSetSize of job (KB)
...
006 (3196552.029.000) 09/28 00:18:22 Image size of job updated: 680640
	562  -  MemoryUsage of job (MB)
	574888  -  ResidentSetSize of job (KB)
...
006 (3196552.029.000) 09/28 00:33:23 Image size of job updated: 680784
	562  -  MemoryUsage of job (MB)
	575032  -  ResidentSetSize of job (KB)
...
006 (3196552.029.000) 09/28 00:38:24 Image size of job updated: 681576
	563  -  MemoryUsage of job (MB)
	575824  -  ResidentSetSize of job (KB)
...
006 (3196552.029.000) 09/28 00:43:24 Image size of job updated: 681936
	563  -  MemoryUsage of job (MB)
	576184  -  ResidentSetSize of job (KB)
...
040 (3196552.029.000) 09/28 00:58:35 Started transferring output files
...
040 (3196552.029.000) 09/28 00:58:35 Finished transferring output files
...
005 (3196552.029.000) 09/28 00:58:35 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 03:12:15, Sys 0 00:00:24  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 03:12:15, Sys 0 00:00:24  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	11709  -  Run Bytes Sent By Job
	1018  -  Run Bytes Received By Job
	11709  -  Total Bytes Sent By Job
	1018  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :    10135        1   2807069 
	   Memory (MB)          :      563     2000      2000 
...
