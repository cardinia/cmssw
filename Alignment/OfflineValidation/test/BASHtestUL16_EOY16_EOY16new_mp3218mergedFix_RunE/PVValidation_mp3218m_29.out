Job started at  Fri Sep 27 21:42:57 CEST 2019
batch dir: /pool/condor/dir_10416 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_277127_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 10M -rw-r--r--. 1 tkello zh 9.9M Sep 28 00:58 PVValidation_mp3218m_277127.root -rw-r--r--. 1 tkello zh 21K Sep 27 21:42 PVValidation_mp3218m_277127_cfg.py -rw-r--r--. 1 tkello zh 11K Sep 28 00:58 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 27 21:43 _condor_stdout -rwxr-xr-x. 1 tkello zh 1018 Sep 27 21:42 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 27 21:43 tkello.cc drwx------. 3 tkello zh 4.0K Sep 27 21:42 tmp drwx------. 3 tkello zh 4.0K Sep 27 21:42 var
Job ended at  Sat Sep 28 00:58:35 CEST 2019
